addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help_it_IT/contents/pscanrules.html

```diff
@@ -136,7 +136,7 @@ <h2 id="id-10010">Cookie HttpOnly</h2>
 Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10010/">10010</a>.
 
 <h2 id="id-90033">Cookie - Loosely Scoped</h2>
-Cookies can be scoped by domain or path. This check is only concerned with domain scope.The domain scope applied to a cookie determines which domains can access it. For example, a cookie can be scoped strictly to a subdomain e.g. <code>www.nottrusted.com</code>, or loosely scoped to a parent domain e.g. <code>nottrusted.com</code>. In the latter case, any subdomain of <code>nottrusted.com</code> can access the cookie. Loosely scoped cookies are common in mega-applications like <code>google.com</code> and <code>live.com</code>.
+Cookies can be scoped by domain or path. This check is only concerned with domain scope. The domain scope applied to a cookie determines which domains can access it. For example, a cookie can be scoped strictly to a subdomain e.g. <code>www.nottrusted.com</code>, or loosely scoped to a parent domain e.g. <code>nottrusted.com</code>. In the latter case, any subdomain of <code>nottrusted.com</code> can access the cookie. Loosely scoped cookies are common in mega-applications like <code>google.com</code> and <code>live.com</code>.
```

Comment:
This should be reverted (it will be overwritten by Crowdin sync). If a space is still needed it should be added to the main file.